Source/Core/Common/MemArenaUnix.cpp

```diff
@@ -117,14 +117,21 @@ LazyMemoryRegion::~LazyMemoryRegion()
   Release();
 }
 
+#if defined __FreeBSD__ || defined __OpenBSD__ || defined __NetBSD__
```

Comment:
Maybe we should also check if MAP_NORESERVE is defined? Just in case it gets implemented in the future.